tests/test_optimizer.py

```diff
@@ -427,6 +427,18 @@ def test_simplify(self):
 """.strip(),
         )
 
+        # uniq_sort for XORs
```

Comment:
These tests should be in the fixture file

Reply:
It seems that MySQL is the only dialect that explicitly supports & parses logical XORs, so I decided for now to keep the parsers as is and isolate these tests in `test_optimizer`. If we want to move these to `simplify.sql`, we'd have to move `exp.Xor` in `parser.py`'s `CONJUCTION_PARSERS` afaict

Reply:
Can't we just annotate the test using `# dialect: mysql`?

Reply:
Oh right, that should work too, moving the tests back to the fixture file